### DISCUSSION TITLE
Recovery Account

### DISCUSSION THREAD

**Discussion by bopfermann** [vercel/community#6395] [Help]
### Summary

I'd like to set up a way for an admin to recovery my account should I get hit by a bus.  How can I do this without paying for another team member for this use case since they will hopefully never need to log in.  I'm not worried about a controlled transfer.  What I'm really concerned about an immediate and unplanned exit in which I couldn't assist in the transfer.  

Is there any way to register a backup that can't login without some formal request/transfer process?

I'm currently signed in via BitBucket, but I don't feel it's appropriate to share my BB credentials for just this purpose.

### Steps to Reproduce

No Steps Known

**Comment by amyegan** [comment]
Hey @bopfermann. The only way you and another person would be able to share ownership of a project is if you were both on the same Pro or Enterprise team. This is usually not a problem since most [commercial projects](https://vercel.com/docs/limits/fair-use-guidelines#commercial-usage) would already be on a Pro team anyway (see [Fair use Guidelines](https://vercel.com/docs/limits/fair-use-guidelines#) for more info).

If it's a non-commercial project where someone else has access to the repo and control of the custom domain, then they would be able to set up their own copy of the site if necessary.
